### PR TITLE
#163388126 Add Travis CI Badge to Readme

### PR DESCRIPTION
**Pull request by Arusey**
**What does this PR do?**
Integrates the Travis CI onto the READme file

**Description of the tasks completed**
Added a markdown for the Travis CI badge on the READme file

**How should this be manually tested?**
After cloning the repo, cd into it and navigate to the READme file to view the raw markdown. Alternatively, you can view the READme file on the this branch to view the badge

**Any Background context you want to provide?**
None

**What are the relevant Pivotal tracker stories?**
#163388126

### CODE REVIEW

File: authors/settings.py [hunk 1]
```diff
@@ -137,7 +137,7 @@
     'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
     'NON_FIELD_ERRORS_KEY': 'error',
 
-    'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
-    ),
+    # 'DEFAULT_AUTHENTICATION_CLASSES': (
```
Comment (sirghiny) [inline authors/settings.py]: Why did you comment this section out?

**Comment by Arusey** [comment]
It was causing an error in the Travis server because it was using a
deprecated Django version. I was waiting for the Django app to be upgraded
first.

**Comment by sirghiny** [comment]
@Arusey Instead of commenting it out, remove the lines of code and restore them once the Django version is upgraded,  it will be cleaner that way since at this level we're not dealing with authentication yet.

**Comment by Arusey** [comment]
Thanks noted

**Pull request merged**